src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -1285,26 +1290,89 @@
                                             TickFrequency="1"
                                             Maximum="300">
-                                        </Slider>
-                                        <TextBlock Text="300%"></TextBlock>
+                                            </Slider>
+                                            <TextBlock Text="{x:Static p:Resources.PreferencesViewZoomScaling300Percent}"></TextBlock>
 
-                                    </StackPanel>
-                                    <StackPanel Grid.Row="2" Width="550">
-                                        <Grid>
-                                            <Grid.ColumnDefinitions>
-                                                <ColumnDefinition Width="125"></ColumnDefinition>
-                                                <ColumnDefinition Width="*"></ColumnDefinition>
-                                            </Grid.ColumnDefinitions>
-                                        </Grid>
-                                        <Label
+                                        </StackPanel>
+                                        <StackPanel Grid.Row="2" Width="550">
+                                            <Grid>
+                                                <Grid.ColumnDefinitions>
+                                                    <ColumnDefinition Width="125"></ColumnDefinition>
+                                                    <ColumnDefinition Width="*"></ColumnDefinition>
+                                                </Grid.ColumnDefinitions>
+                                            </Grid>
+                                            <Label
                                             Name="lblZoomScalingValue"
                                             Width="50"
                                             VerticalAlignment="Center"
                                             HorizontalContentAlignment="Center"
                                             Foreground="{StaticResource PreferencesWindowFontColor}"
                                             FontWeight="Regular"/>
-                                    </StackPanel>
+                                        </StackPanel>
+                                    </Grid>
+                                    <Grid x:Name="PythonZoomScale" Grid.Row="1">
+                                        <Grid.RowDefinitions>
+                                            <RowDefinition Height="Auto"/>
+                                            <RowDefinition Height="Auto"/>
+                                            <RowDefinition Height="Auto"/>
+                                        </Grid.RowDefinitions>
+                                        <StackPanel Margin="0,10,0,5" Grid.Row="0" Orientation="Horizontal">
+                                            <TextBlock  FontSize="12" VerticalAlignment="Center" Text="Python"></TextBlock>
```

Comment:
Can we use `Python Editor` instead of just `Python`, I think it would be more clear?

Reply:
@Jingyi-Wen What do you think?

Reply:
> @Jingyi-Wen What do you think?

Thanks! Good idea!

Reply:
@zeusongit Updated